packages/atomic/src/components/search/result-template-components/atomic-quickview/atomic-quickview.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import defaultResultComponentStory from '../../../../../.storybook/default-result-component-story';
+
+const {defaultModuleExport, exportedStory} = defaultResultComponentStory(
+  'Atomic/ResultList/Quickview',
+  'atomic-quickview',
+  {},
+  {
+    engineConfig: {
+      preprocessRequest: (request) => {
+        const fakeQuery = 'coveo intranet';
```

Comment:
That serves as a way to add some keyword highlighting in the quickview on some PDF that talks about `Coveo for intranet` in our sample org.